src/log.c

```diff
@@ -234,29 +186,46 @@ static int writeEntry(Log *log, raft_entry_t *ety)
     return RR_ERROR;
 }
 
-static int writeHeader(Log *log)
+static raft_entry_t *readEntry(Log *log)
```

Comment:
`writeEntry()` also write the crc of the entry, therefore, `readEntry()` should also read the crc value

Reply:
its extra IO that seems unneccessary (probably not the biggest deal, but didn't serve much point.  we validate it on startup, our use case isn't it getting corrupted during a single runtime (if that would happen, we have worse issues).